build-parent/pom.xml

```diff
@@ -775,6 +775,49 @@
             </build>
         </profile>
 
+        <profile>
+            <id>format-kotlin</id>
+            <activation>
+                <activeByDefault>true</activeByDefault>
+                <property>
+                    <name>!no-format</name>
+                </property>
+                <file>
+                    <exists>src/main/kotlin</exists>
+                </file>
+            </activation>
+            <build>
+                <plugins>
+                    <plugin>
+                        <groupId>com.diffplug.spotless</groupId>
+                        <artifactId>spotless-maven-plugin</artifactId>
+                        <version>2.25.0</version>
+                        <executions>
+                            <execution>
+                                <id>format-kotlin</id>
+                                <phase>process-sources</phase>
+                                <goals>
+                                    <goal>apply</goal>
+                                </goals>
+                            </execution>
+                        </executions>
+
+                        <configuration>
+                            <kotlin>
+                                <!--
+                                    <ktfmt>
+                                        <style>DEFAULT</style> &lt;!&ndash; optional, other options are DEFAULT, DROPBOX, GOOGLE and KOTLINLANG &ndash;&gt;
+                                    </ktfmt>
+                                -->
+                                <ktlint/>
+
+                            </kotlin>
+                        </configuration>
+                    </plugin>
+                </plugins>
+            </build>
+        </profile>
```

Comment:
Do we really need a different profile? Would it not be better to use the existing profiles?

Reply:
I'm ok combining them but this one also looks for the existence of kotlin sources to activate.  So it's not trying to do work most of the time.  It might be a microoptimization but it made sense at the time.  @gsmet?